components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AMDefaultKeyManagerImpl.java

```diff
@@ -339,16 +339,20 @@ public AccessTokenInfo getNewApplicationAccessToken(AccessTokenRequest tokenRequ
                 //Revoke the Old Access Token
                 httpRevokePost.setEntity(new UrlEncodedFormEntity(revokeParams, "UTF-8"));
                 int statusCode;
+                String responseBody;
                 try {
                     HttpResponse revokeResponse = executeHTTPrequest(revokeEndpointPort, revokeEndpointProtocol, 
                                                                      httpRevokePost);
                     statusCode = revokeResponse.getStatusLine().getStatusCode();
+                    responseBody = EntityUtils.toString(revokeResponse.getEntity());
                 } finally {
                     httpRevokePost.reset();
                 }
 
                 if (statusCode != 200) {
```

Comment:
Isn't it better to use HttpStatus.SC_OK  constant instead of 200 ?

Reply:
We can do that will imprive the code